### ISSUE TITLE
response inconsistency in createBankTransactionAttachmentByFileName

### ISSUE THREAD

**Issue by ifeelfishy** [XeroAPI/xero-node#430]
**SDK you're using (please complete the following information):**
 - Version [4.7.2]

**Describe the bug**
posting attachment to bank transaction with
```
const outcome = await xero.accountingApi.createBankTransactionAttachmentByFileName(...)
```
results in  `outcome.body`
```
body: Attachments { attachments: undefined }
```
however `outcome.response.body` contains the necessary information

**To Reproduce**
call  `await xero.accountingApi.createBankTransactionAttachmentByFileName(...)`

**Expected behavior**
`outcome.body` should contain the attachment information

**Comment by SerKnight** [comment]
Thanks @ifeelfishy - this is a bug in the deserialize function `public static deserialize(data: any, type: string) {` the attachments are not being mapped correctly to their corresponding object.

We are working on this and will update here asap!

**Comment by SerKnight** [comment]
FYI - We've found the culprit but fix is going to take a bit more testing to validate across all the file attachment functions.

**Comment by RettBehrens** [comment]
@ifeelfishy this has been resolved in xero-node 4.7.5 and an example call added to the [sample app](https://github.com/XeroAPI/xero-node-oauth2-app/blob/f1fbd3a08e840e54e8ce57f7050ddde6686208d8/src/app.ts#L447)
